tests/subscriptions/test_executor_consumer.py

```diff
@@ -155,13 +155,14 @@ def generate_message(
 def test_execute_query_strategy() -> None:
     state.set_config("executor_sample_rate", 1.0)
     dataset = get_dataset("events")
+    entity_names = ["events"]
     max_concurrent_queries = 2
     executor = ThreadPoolExecutor(max_concurrent_queries)
     metrics = TestingMetricsBackend()
     next_step = mock.Mock()
 
     strategy = ExecuteQuery(
-        dataset, executor, max_concurrent_queries, metrics, next_step
+        dataset, entity_names, executor, max_concurrent_queries, metrics, next_step
```

Comment:
please let's test the negative case when a query is not executed because it is not in the list of entity names provided.

Reply:
Sounds good, added the test